mvp-spec/partials/openrtb-request-with-transmissions.json

```diff
@@ -0,0 +1,88 @@
+{
+    "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+    "at": 1, 
+    "cur": [ "USD" ],
+    "imp": [
+        {
+            "id": "1",
+            "bidfloor": 0.03,
+            "banner": {
+                "h": 250,
+                "w": 300,
+                "pos": 0
+            },
+            "ext": {
+                "prebid_sso": {
+                    "version": 0,
+                    "seed": {
+                        "version": 0,
+                        "transaction_id": 1234567,
+                        "publisher": "publisher.com",
+                        "source": {
+                            "domain": "publisher0.com",
+                            "timestamp": 1639589531,
+                            "signature": "12345_signature"
+                        }
+                    },
+                    "parents": [],
```

Comment:
Explained briefly in the `ad-server-implementation.md` but here is a highlight of the result of removing duplicated Prebid SSO Data in the impression (`imp`).